loadtests/loadtest/calls.py

```diff
@@ -5,12 +5,13 @@
 from requests_hawk import HawkAuth
 
 from fxa.core import Client
-from fxa.tests.utils import TestEmailAccount
+from fxa import errors
 from fxa.plugins.requests import FxABrowserIDAuth
 
 # XXX Using the same fxa as the dev server, as the staging setup isn't working
 # for me.
-DEFAULT_FXA_URL = "https://api.accounts.firefox.com/v1"
+DEFAULT_FXA_URL = "https://api-accounts.stage.mozaws.net"
+ERROR_ACCOUNT_EXISTS = 101
```

Comment:
It's a shame you can't import this as a constant from the `fxa` module; something for us to think about in a future enhancement